atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -140,6 +142,10 @@ private SweepResults doRun(TableReference tableRef,
                                byte[] startRow,
                                RunType runType,
                                Sweeper sweeper) {
+        log.info("Beginning iteration of sweep for table {} starting at row {}",
```

Comment:
If we have this logging, do we want to remove the loggings on BackgroundSweeperImpl - the ones on getTableToSweep? Otherwise we'd log the same thing twice (that we're starting to sweep table {} on row {}).

You also probably want to update the sweep-logs.rst document to keep track of where we log this line.

Reply:
oh i see, we already having logging there. ok, i'll just move this to the `SweeperService`

Reply:
actually, going to leave this  and delete the other, as otherwise we'd have to log in multiple places